app/controllers/comments_controller.rb

```diff
@@ -33,6 +34,11 @@ def create
       if @comment.save
         format.html { redirect_back fallback_location: root_path, notice: "Comment was successfully created." }
         format.json { render :show, status: :created, location: @comment }
+        # format.js do 
```

Comment:
If you wanted to be super-picky you could delete this commented-out code, just to keep things clean!